open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/static/src/subject-area/subject-area-view.js

```diff
@@ -16,6 +16,8 @@ import "../../node_modules/@polymer/paper-dialog/paper-dialog.js";
 import "../../node_modules/@polymer/paper-dialog-behavior/paper-dialog-behavior.js";
 import "../../node_modules/@polymer/iron-localstorage/iron-localstorage.js";
 import "../../node_modules/@polymer/iron-ajax/iron-ajax.js";
+import {mixinBehaviors} from "../../node_modules/@polymer/polymer/lib/legacy/class.js";
```

Comment:
paths should start with @polymer... or @vaadin...
`"../../node_modules/@polymer/polymer/lib/legacy/class.js";`

should be
`'@polymer/polymer/lib/legacy/class.js';`